tests/testthat/test-enw_quantiles_to_long.R

```diff
@@ -7,4 +7,15 @@ test_that("enw_quantiles_to_long can manipulate posterior draws as expected", {
         c("rhat", "ess_bulk", "ess_tail") := NULL][!is.na(variable)][]
     )
   )
-})
\ No newline at end of file
+})
+
+test_that("enw_quantiles_to_long uses data.table::melt not deprecated reshape2::melt", {
```

Comment:
Being a bit pedantic, but: the test here is currently that `enw_quantiles_to_long` can receive a data.frame and still work - that's a perfectly good test, as we don't actually care what it does internally.